### DISCUSSION TITLE
Settings to handle bail out shortcut command

### DISCUSSION THREAD

**Discussion by lycanhunt3r** [monkeytypegame/monkeytype#4896] [Ideas]
Add options to handle the shift+enter bail command in the settings: e.g. off | double shift+enter | shift+enter and manual click on popup.

My current predicament is that I get some novel chapters a day to type in the "custom" section, but when a part of the text is as following:

{{{
And so, character said:

"Some dialogue."
}}}

I end up pressing the shift to add the two dots, double enter, and then keep shift pressed to type the quotes (") and the first upper character. Which, consequently, bails out of the test immediately. I looked around but there doesn't seem to have any options about it nor there are any solutions available. Is it an unavailable feature, or can I resolve it in some way that I didn't find?

**Comment by Miodec** [comment]
There is no setting at the moment which changes the behavior of the bail out shortcut im afraid.
